modules/Campaigns/language/en_us.lang.php

```diff
@@ -149,7 +149,7 @@
     'LBL_CAMPAIGN_FREQUENCY' => 'Frequency:',
     'LBL_NEWSLETTERS' => 'View Newsletters',
     'LBL_NEWSLETTER' => 'Newsletter',
-    'LBL_NEWSLETTER_FORENTRY' => 'NewsLetter',
```

Comment:
This is use in code so it would likely cause issues removing it or even translating it. The code would need to be address not to read the value from here then it should be omitted.

Reply:
@mattlorimer its strange... i didnt removed the string     'LBL_NEWSLETTER_FORENTRY' => 'NewsLetter',  even if it shows as deleted in Github interface.
I just fixed the spelling.
`    'LBL_NEWSLETTER_FORENTRY' => 'NewsLetter',`
to 
`    'LBL_NEWSLETTER_FORENTRY' => 'Newsletter',`
Editing the file (even in github web interface) shows the string there with the new spelling!
Maybe you find a way to fix this!

Reply:
I see it now... if you change a single letter in a word, Github web interface shows a string as deleted (red) and then added back in the next line (green). You missed the added one, the line below.